signalfx-java/src/main/java/com/signalfx/signalflow/ServerSentEventsTransport.java

```diff
@@ -0,0 +1,531 @@
+/*
+ * Copyright (C) 2016 SignalFx, Inc. All rights reserved.
+ */
+package com.signalfx.signalflow;
+
+import java.io.BufferedReader;
+import java.io.Closeable;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.UnsupportedEncodingException;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.NoSuchElementException;
+import java.util.regex.Pattern;
+
+import org.apache.http.HttpEntity;
+import org.apache.http.NameValuePair;
+import org.apache.http.StatusLine;
+import org.apache.http.client.config.RequestConfig;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.client.utils.URIBuilder;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.conn.BasicHttpClientConnectionManager;
+import org.apache.http.message.BasicNameValuePair;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.signalfx.connection.AbstractHttpReceiverConnection;
+import com.signalfx.endpoint.SignalFxEndpoint;
+
+/**
+ * Server-Sent Events transport.
+ * 
+ * Implements a transport to the SignalFlow API that uses simple HTTP requests and reads Server-Sent
+ * Events streams back from SignalFx. One connection per SignalFlow computation is required when
+ * using this transport. This is a good transport for single, ad-hoc computations. For most use
+ * cases though, the WebSocket-based transport is more efficient and has lower latency.
+ * 
+ * @author dgriff
+ */
+public class ServerSentEventsTransport implements SignalFlowTransport {
```

Comment:
where do you see the jetty client for sse?  I see eventsourceservlet for jetty 9 (later than the jetty 8 with java 1.6 support) but that is not useful for a java client perspective.

The impl in this PR uses the existing apache httpclient with proxy support and should implement the sse spec correctly and is pretty minimal is size.